pyth-sdk-terra/README.md

```diff
@@ -1,83 +1,44 @@
-
-```rust
-let price_feed: PriceFeed = query_price_feed(deps.querier, contract_addr, id)?.price_feed;
 ```
-
-### Example contract
-
-Checkout [Example Terra Contract](../examples/terra-contract/) as an example which uses Pyth Terra contract to fetch a price.
-
-## Price Feed and Price API
-
-The `PriceFeed` struct contains several useful functions for working with the price.
-Some of these functions are described below.
-For more detailed information, please see the crate documentation.
-
-### Get the current price
-
-Read the current price from a `PriceFeed`: 
-
-```rust
-let current_price: Price = price_feed.get_current_price().ok_or(StdError::not_found("Current Price is not available"))?;
-println!("price: ({} +- {}) x 10^{}", current_price.price, current_price.conf, current_price.expo);
+[dependencies]
+pyth-sdk-terra = { version = "<current version>" }
 ```
 
-The price is returned along with a confidence interval that represents the degree of uncertainty in the price.
-Both values are represented as fixed-point numbers, `a * 10^e`. 
-The method will return `None` if the price is not currently available.
-
-### Non-USD prices 
-
-Most assets in Pyth are priced in USD.
-Applications can combine two USD prices to price an asset in a different quote currency:
-
-```rust
-let btc_usd: Price = ...;
-let eth_usd: Price = ...;
-// -8 is the desired exponent for the result 
-let btc_eth: Price = btc_usd.get_price_in_quote(&eth_usd, -8);
-println!("BTC/ETH price: ({} +- {}) x 10^{}", price.price, price.conf, price.expo);
-```
+See [pyth-sdk-terra on crates.io](https://crates.io/crates/pyth-sdk-terra) to get the most recent version.
 
-### Price a basket of assets
+## Usage
 
-Applications can also compute the value of a basket of multiple assets:
+Simply call the `query_price_feed` function in your Terra contract with a price feed id:
 
 ```rust
-let btc_usd: Price = ...;
-let eth_usd: Price = ...;
-// Quantity of each asset in fixed-point a * 10^e.
-// This represents 0.1 BTC and .05 ETH.
-// -8 is desired exponent for result
-let basket_price: Price = Price::price_basket(&[
-    (btc_usd, 10, -2),
-    (eth_usd, 5, -2)
-  ], -8);
-println!("0.1 BTC and 0.05 ETH are worth: ({} +- {}) x 10^{} USD",
-         basket_price.price, basket_price.conf, basket_price.expo);
+// Pyth network testnet contract address
+pyth_contract_addr: string = "terra1hdc8q4ejy82kd9w7wj389dlul9z5zz9a36jflh";
+// Price feed id for BTC/USD on testnet
+price_feed_id: string = "0xf9c0172ba10dfa4d19088d94f5bf61d3b54d5bd7483a322a982e1373ee8ea31b";
+
+let price_feed: PriceFeed = query_price_feed(deps.querier, pyth_contract_addr, price_feed_id)?.price_feed;
+let current_price: Price = price_feed.get_current_price().ok_or(StdError::not_found("price is not currently available"))?;
+println!("current BTC/USD price: ({} +- {}) x 10^{}", current_price.price, current_price.conf, current_price.expo);
```

Comment:
made this example complete so that the reader can basically copy-paste it into their code